bin/sdk_gen

```diff
@@ -214,11 +189,10 @@ const regen = async (release) => {
     const branch = branchActive()
     if (await pullci(release)) {
       return batch([
-        (release) => releasePlease(release),
         'yarn gen',
         'bin/smoke typescript python kotlin',
         'git add -A',
-        `git commit -m "feat: generate SDKs for Looker ${release}"`,
+        `git commit -m "feat: generate SDKs for Looker ${release}" -m "Release-As: ${release}.0`,
```

Comment:
@jkaster This additional body will tell release-please to calendar version appropriately.

Reply:
Thanks for the reminder. You did tell me that before.

Reply:
TIL you can use `-m` twice in a git commit